src/plugins/irc-events/link.js

```diff
@@ -390,7 +390,7 @@ function fetch(uri, headers) {
 		try {
 			const gotStream = got.stream(uri, {
 				retry: 0,
-				timeout: 5000,
+				timeout: Helper.config.prefetchTimeout, // milliseconds
```

Comment:
Can you add a warning and fallback for if `prefetchTimeout` isn't present? 

Something like this a few lines up:
```js
const prefetchTimeout = Helper.config.prefetchTimeout;
if (!prefetchTimeout) {
    console.warn("prefetchTimeout is missing from your The Lounge configuration, defaulting to 5000");
}
```

Reply:
Take a glance at the new commits and see if those changes suit?